Sources/Classes/core/Protocols/MiniAppMessageDelegate.swift

```diff
@@ -10,6 +10,12 @@ public protocol MiniAppMessageDelegate: MiniAppUserInfoDelegate, MiniAppShareCon
     /// Interface that should be implemented to return alphanumeric string that uniquely identifies a device.
     func getUniqueId(completionHandler: @escaping (Result<String?, MASDKError>) -> Void)
 
+    /// Interface that should be implemented to return alphanumeric string that uniquely identifies a device.
+    func getContactId(completionHandler: @escaping (Result<String?, MASDKError>) -> Void)
+
+    /// Interface that should be implemented to return alphanumeric string that uniquely identifies a device.
```

Comment:
Same description 🤔 , the purchase is same. However do you think we can something extra here ?

Reply:
updated a slight difference do you think I should add more info about the assembly of the "id's"?

Reply:
Looks ok now